third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -9,6 +9,18 @@ use std::path::Path;
 
 datatest_stable::harness!(run, TEST_DIR, r".*\.move$");
 
+/// Root of tests which require to skip v1-v2 comparison
+const SKIP_V1_COMPARISON_PATH: &str = "/no-v1-comparison/";
+
 fn run(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
-    vm_test_harness::run_test_with_config(TestRunConfig::ComparisonV1V2, path)
+    let test_config: TestRunConfig = if let Some(p) = path.to_str() {
```

Comment:
I am not sure what our policy is on using features not available by default, but using let-chains would reduce some code duplication here:

`if let Some(p) = path.to_str() && p.contains(...) {...} else {...}`

However, to get this to work, we would need to add `#![feature(let_chains)]` above.

Reply:
I prefer not to do that, since it seems to be unstable.  How about. .unwrap_or_default() instead of the let match.

Reply:
This was just a nit, and you are right that this is marked as "unstable feature". I don't have an opinion regarding `unwrap_or_default`.